packages/create-sitecore-jss/src/templates/nextjs-sxa/src/Layout.tsx

```diff
@@ -24,16 +23,15 @@ interface RouteFields {
   [key: string]: unknown;
   Title: Field;
 }
- 
+
 const Layout = ({ layoutData }: LayoutProps): JSX.Element => {
   const { route } = layoutData.sitecore;
-
   const fields = route?.fields as RouteFields;
 
   return (
     <>
       <Head>
-        <title>{fields?.Title.value.toString() || 'Page'}</title>
+        <title>{fields?.Title?.value.toString() || 'Page'}</title>
```

Comment:
`RouteFields` interface contains `Title` as required, in this case it should be optional there or required here